client/src/components/App/Content/LandingPage/InvestigationTable/TableFilter/useTableFilter.ts

```diff
@@ -0,0 +1,59 @@
+import { format } from 'date-fns';
+import { useEffect, useState } from 'react';
+
+import { TimeRange } from 'models/TimeRange';
+import timeRanges from 'models/enums/timeRanges';
+
+const rangeError = 'טווח תאריכים צריך להיות תקין';
+
+const useTableFilter = (props : Props) => {
+    
+    const {timeRangeFilter , onTimeRangeFilterChange} = props;
+    const customTimeRange = timeRanges[4];
```

Comment:
maybe export 4 to const

Reply:
changed the variable to be a variable exported from enums/timeRanges.ts because I'm creating this const in serval places